src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -2732,6 +2744,23 @@ public Task<RedisValue> StringSetRangeAsync(RedisKey key, long offset, RedisValu
             _ => throw new ArgumentException("Expiry time must be either Utc or Local", nameof(when)),
         };
 
+        private Message GetCopyMessage(in RedisKey source, RedisKey destination, int database, bool replace, CommandFlags flags)
+        {
+            if (database < -1) throw new ArgumentOutOfRangeException(nameof(database));
+            else if (database == -1)
+            {
+                return replace?
+                    Message.Create(Database, flags, RedisCommand.COPY, source, destination, RedisLiterals.REPLACE):
+                    Message.Create(Database, flags, RedisCommand.COPY, source, destination);
+            }
+            else
+            {
+                return replace?
+                    Message.Create(Database, flags, RedisCommand.COPY, source, destination, database, RedisLiterals.REPLACE):
+                    Message.Create(Database, flags, RedisCommand.COPY, source, destination, database);
+            }
```

Comment:
Can use a `switch` expression to tidy here now that we have conditionals - see `GetStringSetMessage` as an example :)